pyproject.toml

```diff
@@ -27,6 +27,8 @@ python = "^3.9"
 giant-mixins = "*"
 django-filer = "*"
 swapper = "^1.3.0"
+flake8 = "4.0.1"
+isort = "^5.10.1"
```

Comment:
I would say these shouldn't be in the main deps as they will get installed into main deps in projects when this is installed

Reply:
yes ofc sorry did that last time i remember now